avaframe/ana3AIMEC/dfa2Aimec.py

```diff
@@ -117,13 +139,22 @@ def mainDfa2Aimec(avaDir, cfgSetup):
     fU.makeADir(massDir)
     log.info('Aimec Work folders created to start postprocessing com1DFA data')
 
-    # Setup input from com1DFA and export to Work ana3AIMEC
-    suffix = {'type' : ['pfd', 'ppr', 'pfv'], 'directory' : ['dfa_depth', 'dfa_pressure', 'dfa_speed']}
-    for suf, dir in zip(suffix['type'], suffix['directory']):
-        fU.getDFAData(avaDir, workDir, suf, dir)
-
     # Write the paths to this files to a file
     writeAimecPathsFile(cfgSetup, avaDir)
 
-    # Extract the MB info
-    extractMBInfo(avaDir)
+    comModules = cfgSetup['comModules'].split('|')
+    countStart = 0
+    countFile = 0
```

Comment:
What is the purpose of the counting done here?

Reply:
this is required as the files for aimec are renamed to 000001.txt, 000002.txt, etc and data from two modules is compared the function needs to know where to start the counter

Reply:
removed